### ISSUE TITLE
Error for metaclass conflict could be improved

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#6176]
When a metaclass conflict occurs, the current error message doesn't provide any details about which metaclasses are conflicting.

```python
class Meta1(type): ...
class Base1(metaclass=Meta1): ...

class Meta2(type): ...
class Base2(metaclass=Meta2): ...

class Child(Base1, Base2): ... # Error: metaclass conflict
```

It would be helpful if pyright included the conflicting metaclasses to help in the diagnosis of the error.

**Comment by erictraut** [comment]
The message now includes the metaclasses:

![image](https://github.com/microsoft/pyright/assets/7040122/f8d573f6-0952-418c-9965-785fcac05834)

**Comment by erictraut** [comment]
This is included in pyright 1.1.332, which I just published. It will also be included in a future release of pylance.
